.github/workflows/snapshot-publish.yml

```diff
@@ -2,6 +2,8 @@ name: Publish snapshot of test scan
 
 env:
   CD_DETECTOR_EXPERIMENTS: 1
+  PipReportSkipFallbackOnFailure: "true"
+  PIP_INDEX_URL: "https://pypi.python.org/simple"
```

Comment:
The snapshot reference for the verification tests needs to be updated with the same environment as the verification pipeline or otherwise it will fail.